asplib_modules/FFT/CMakeLists.txt

```diff
@@ -21,15 +14,17 @@ file (GLOB_RECURSE ASPLIB_FFT_HEADERS
 
 asplib_source_group("${ASPLIB_FFT_SOURCES}")
 asplib_source_group("${ASPLIB_FFT_HEADERS}")
+
 add_library(asplib_FFT STATIC ${ASPLIB_FFT_SOURCES} ${ASPLIB_FFT_HEADERS})
+add_library(asplib::FFT ALIAS asplib_FFT)
```

Comment:
The alias target can be used within the project and ensures with its :: name that target_link_libraries fails already at configure time when the target is not here. Without that, cmake would just assume that it's a system lib and just adds -llib to the linker flag. so you see errors only at link time.